packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -48,7 +48,10 @@ export async function init() {
     database,
     senders.email
   );
+  const statsd = Container.get(StatsD);
```

Comment:
Do we know that `StatsD` is already set?

Reply:
Yep, that's done above in the `setupProcesingTaskObjects` method.